Library/Model/SPIMessage.m

```diff
@@ -133,6 +151,8 @@ - (instancetype)initWithMessageId:(NSString *)mid
         _eventName = eventName.copy;
         _data = data.copy;
         _needsEncryption = needsEncryption;
+        _connID = 0;
```

Comment:
In Objective-C does it matter if elsewhere this is referred to as a string?

Reply:
Objective-c is pretty much forgiving in this matter. I have changed this everywhere but it seem I missed this part of code